.mailmap

```diff
@@ -568,8 +624,10 @@ Tim Diekmann <t.diekmann.3dv@gmail.com>
 Tim Hutt <tdhutt@gmail.com>
 Tim JIANG <p90eri@gmail.com>
 Tim Joseph Dumol <tim@timdumol.com>
+Tim Neuman <timnn@google.com> <mail@timnn.me>
```

Comment:
Could you please swap these around so that `mail@timnn.me` is the primary address?

Also, theres an "n" missing in `Neumann`.

Otherwise LGTM.